packages/wp-now/README.md

```diff
@@ -179,18 +179,17 @@ Run `wp-now start --blueprint=path/to/blueprint-example.json` where `blueprint-e
 		{
 			"step": "defineWpConfigConsts",
 			"consts": {
-				"WP_DEBUG": true,
-				"WP_DEBUG_LOG": true
-			},
-			"method": "define-before-run"
+				"WP_DEBUG": true
+			}
 		}
 	]
 }
 ```
 
-This will enable the debug logs and will create a `debug.log` file in the `~/.wp-now/wp-content/${project}/debug.log` directory.
+Because we didn't define a method for `defineWpConfigConsts`, and the default method is `rewrite-wp-config`, this blueprint will update your `~/.wp-now/wordpress-versions/latest/wp-config.php` enabling the WP_DEBUG constant and will create a `debug.log` file in the `~/.wp-now/wp-content/${project}/debug.log` directory.
+The next time you execute `wp-now start` in any project, the variable `WP_DEBUG` will still be set to true.
 
-If you prefer to set a custom path for the debug log file, you can set `WP_DEBUG_LOG` to be a directory. Remember that the `php-wasm` server runs udner a VFS (virtual file system) where the default documentRoot is always `/var/www/html`.
+If you prefer to set a custom path for the debug log file, you can set `WP_DEBUG_LOG` to be a directory. Remember that the `php-wasm` server runs udner a VFS (virtual file system) where the default documentRoot for wp-now is always `/var/www/html`.
```

Comment:
```suggestion
If you prefer to set a custom path for the debug log file, you can set `WP_DEBUG_LOG` to be a directory. Remember that the `php-wasm` server runs under a VFS (virtual file system) where the default documentRoot for wp-now is always `/var/www/html`.
```

Reply:
Thanks for the suggestion on the typo.

Yes, I agree that it's odd and it would be awesome to keep the wp version intact 🤔 .
We can probably copy the whole wp directory in VFS and do not mount it, to avoid conflicts.

Anyway, this PR only document the current behaviour.